jobs/fatal_od.py

```diff
@@ -49,8 +50,13 @@ def stringify_datetimes(self, in_data):
 package_id = '945f9505-f33b-46e1-9c43-6c3315b4b0cd'
 resource_name = 'Fatal Accidental OD 2014'
 
+target = 'accidental_fatal_overdoses/fatal_od_mock.csv'
+
+with open(HERE + '/../settings.json') as f:
```

Comment:
I think the pipeline should have access to the configuration directly, so you shouldn't have to do this.